developing/README.md

```diff
@@ -0,0 +1,189 @@
+  # set this to 8000 for development.
+  PORT = environ.get('PROXSTAR_PORT', '5000')
+
+  # The name of your proxstar server. This matters for authenticating with CSH
+  # SSO, so change this to localhost:8000
+  SERVER_NAME = environ.get('PROXSTAR_SERVER_NAME', 'proxstar.csh.rit.edu')
+
+  # Secret key for authenticating with SSO.
+  # Change this to literally anything, just don't leave it blank.
+  SECRET_KEY = environ.get('PROXSTAR_SECRET_KEY', '')
+
+  # OIDC
+
+  # Leave all of this alone.
+  OIDC_ISSUER = environ.get('PROXSTAR_OIDC_ISSUER', 'https://sso.csh.rit.edu/auth/realms/csh')
+  OIDC_CLIENT_CONFIG = {
+      'client_id': environ.get('PROXSTAR_CLIENT_ID', 'proxstar'),
+      'client_secret': environ.get('PROXSTAR_CLIENT_SECRET', ''), # Just kidding, talk to an RTP to get this.
+      'post_logout_redirect_uris': [
+          environ.get('PROXSTAR_REDIRECT_URI', 'https://proxstar.csh.rit.edu/logout')
+      ],
+  }
+
+  # Proxmox
+
+  # Your list of proxmox hosts. You only need one for development.
+  PROXMOX_HOSTS = [host.strip() for host in environ.get('PROXSTAR_PROXMOX_HOSTS', '').split(',')]
+  # Username and group of your test user. For example, 'proxstartest@pam'
+  PROXMOX_USER = environ.get('PROXSTAR_PROXMOX_USER', '')
+  # Said user's password
+  PROXMOX_PASS = environ.get('PROXSTAR_PROXMOX_PASS', '')
+  # Location of ISO storage on your server.
+  PROXMOX_ISO_STORAGE = environ.get('PROXSTAR_PROXMOX_ISO_STORAGE', 'nfs-iso')
+  # Username of SSH user (probably the same)
+  PROXMOX_SSH_USER = environ.get('PROXSTAR_PROXMOX_SSH_USER', '')
+  # Paste that SSH key I told you to generate.
+  PROXMOX_SSH_KEY = environ.get('PROXSTAR_PROXMOX_SSH_KEY', '')
+  # If you put a password on it, then paste that here.
+  PROXMOX_SSH_KEY_PASS = environ.get('PROXSTAR_PROXMOX_SSH_KEY_PASS', '')
+
+  # STARRS
+
+  # The IP address or hostname of your STARRs host.
+
+  # Since you should be hosting this in a container, make it 127.0.0.1
+  STARRS_DB_HOST = environ.get('PROXSTAR_STARRS_DB_HOST', '')
+
+  # The name of your STARRS DB
+  # It.... it should be STARRS.
+  STARRS_DB_NAME = environ.get('PROXSTAR_DB_NAME', 'starrs')
+
+  # The username of your STARRS DB
+  # I just used the postgres user and it seemed to work so uhhhhhhhhhhh
+  STARRS_DB_USER = environ.get('PROXSTAR_DB_USER', '')
+
+  # Password for Postgres user
+  # (You configure this when setting up the Postgres container just use that PWord)
+  STARRS_DB_PASS = environ.get('PROXSTAR_DB_PASS', '')
+
+  # STARRS username
+  # Leave this alone.
+  STARRS_USER = environ.get('PROXSTAR_STARRS_USER', 'proxstar')
+
+  #???
+  # IDK leave this alone, too.
+  STARRS_IP_RANGE = environ.get('PROXSTAR_IP_RANGE', '')
+
+  # LDAP
+
+  # Ask an RTP about this. You need them to authenticate. They're somewhere on OKD, probably.
+  LDAP_BIND_DN = environ.get('PROXSTAR_LDAP_BIND_DN', '')
+  LDAP_BIND_PW = environ.get('PROXSTAR_LDAP_BIND_PW', '')
```

Comment:
How much ldap access does proxstar need? If it's nothing special, a users's creds will work.